resources/views/list.blade.php

```diff
@@ -46,6 +54,10 @@
 	<input id="tableFilter_{{ $entityType }}" type="text" style="width:180px;margin-right:17px;background-color: white !important"
         class="form-control pull-left" placeholder="{{ trans('texts.filter') }}" value="{{ Input::get('filter') }}"/>
 
+	@if (Utils::isSelfHost())
+    	@stack('top_right_buttons')
+    @endif
```

Comment:
Indentation error? :)

Reply:
Weird, my local copy doesn't have the indentation problem.

Reply:
Looks like it was an issue with tabs versus spaces.